1.9/usage/cli/command-reference/dcos.md

```diff
@@ -0,0 +1,33 @@
+---
+post_title: dcos
+menu_order: 0
+---
+
+# Description
+Manage DC/OS environment variables. 
+
+# Usage
+
+``` bash
+dcos [options] [<command>] [<args>...]
+```
+
+# Options
+
+| Name, shorthand | Default | Description |
+|---------|-------------|-------------|
+| `--debug`   |             |  Enable debug mode. |
+| `--help, h`   |             |  Print usage. |
+| `--log-level=<log-level>`  |             | Set the logging level. This setting does not affect the output sent to stdout.  |
+|  `--version, v`  |             |  Print version information.  |
+
+`<log-level>`
+The severity levels are:
+
+* debug    Prints all messages.
+* info     Prints informational, warning, error, and critical messages.
+* warning  Prints warning, error, and critical messages.
+* error    Prints error and critical messages.
+* critical Prints only critical messages to stderr.
```

Comment:
I don't know how often this is the case, but I learned today that you need to but the log level in quotation marks, e.g., `dcos --log-level="debug" package install hdfs`. Not sure if or how that can be communicated here